repo/incubating/flink/versions/0/flinkapplication-frameworkversion.yaml

```diff
@@ -21,7 +21,6 @@ spec:
     - name: JOB_ARGUMENTS
       description: Arguments to pass to Job at runtime
       default: ""
-      trigger: restart
```

Comment:
was removed as this lead to the behavior of when a job was resumed to immediately after 1s be cancelled again.